back/index.php

```diff
@@ -1,3 +1,19 @@
 <?php
-    echo 'hello';
+
+// Traitement erreur et sécurité
+error_reporting(-1);
+ini_set('display_errors', 'On');
+set_error_handler("var_dump");
+define ('CONST_INCLUDE', NULL);
+
+require_once path(__DIR__ . '/../App/Connexion/Connexion.php');
```

Comment:
Bad location for index.php, put './App/...' or move index into App directory and write './Connexion/...'